stan/math/opencl/kernels/add.hpp

```diff
@@ -40,7 +40,43 @@ static const char *add_kernel_code = STRINGIFY(
  */
 const global_range_kernel<cl::Buffer, cl::Buffer, cl::Buffer, int, int> add(
     "add", {indexing_helpers, add_kernel_code});
+// \cond
+static const char *add_batch_kernel_code = STRINGIFY(
+    // \endcond
+    /**
+     * Adds all the matrices in a batch
```

Comment:
Can you say a little more about this? Is this basically a sum function over a collection of matrices?

Reply:
Yes, A contains a batch of K smaller matrices of size NxM. This kernel adds elements at matching indices into a resulting NxM matrix.

Reply:
Added to doxygen.